modules/client-common/src/test/java/org/apache/ignite/internal/client/proto/ClientComputeJobPackerUnpackerTest.java

```diff
@@ -78,6 +84,13 @@ private static List<Object> pojo() {
         return List.of(Pojo.generateTestPojo());
     }
 
+    private static List<Arguments> notMarshalled() {
+        return List.of(
+                arguments(Tuple.create(), ComputeJobType.MARSHALLED_TUPLE),
```

Comment:
Shall we introduce a couple of new `ComputeJobType`s? The idea behind `MARSHALLED` prefix is that is *was really marshalled*. But here we do not marshal the argument.

Reply:
I'm not sure if `MARSHALLED_` prefix is necessary. 
* All types of job args and results can be in marshalled or unmarshalled state
* A job can take a tuple and return a primitive, etc, so this is not a job type, but a job argument/result type

Reply:
Maybe you are right. My idea was to add more information to the argument/result like "was this thing marshalled or not?". The answer to this question might be handy but most likely it will not.

Reply:
Marshalling, packing, serialization - those are all synonyms in our case, I think. Or is there a difference?

> drop the MARSHALLED_ prefix for the tuple and pojo types

Agree.